tutorials/bigtable-stackdriver/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Using matplotlib to visualize Stackdriver metrics for Bigtable
```

Comment:
Title should use title case (and the correct product names): "Using matplotlib to Visualize Stackdriver Monitoring Metrics for Cloud Bigtable"